subgraphs/_reference_/src/common/rewards.ts

```diff
@@ -0,0 +1,198 @@
+//////////////////////////
+// AUTHOR: CHRIS STEEGE //
```

Comment:
Can you remove this? People can find the author with `git blame`. Also version is not needed for a helper lib

Reply:
Yeah - I removed this file from this pull request. I'll change it in the other.